packages/react-native-web/src/exports/Dimensions/index.js

```diff
@@ -77,13 +77,24 @@ export default class Dimensions {
     }
 
     const win = window;
-    const docEl = win.document.documentElement;
+    let height;
+    let width;
+
+    if (win.visualViewport) {
```

Comment:
Is this the same as "if safari mweb"?

Reply:
Maybe we should add a comment briefly explaining this if/else?

Reply:
> Is this the same as "if safari mweb"?

I think it's a feature check, but according to caniuse all live browsers support the `visualViewport`: https://caniuse.com/?search=visualViewport

Since react-native-web also dropped support for IE11, I don't think we need to check `if (win.visualViewport)` - it will always be `true` 

--- 

Same for the other case that assigns the listener

Reply:
Agree it would be good to add a comment. Also not sure about support IE I agree with @kidroca and don't think it matters.

Reply:
Oh actually if the maintainer is asking for it here then I'd say we just roll with this: https://github.com/necolas/react-native-web/pull/2438#issuecomment-1325575727

Reply:
@marcaaron is correct. The feature check is not required as all modern browsers support VisualViewport which is evident from the above caniuse.com link.

But the original repo maintainer requested that check so I had to add it.

Reply:
I will add a comment summarizing of my changs today. Sorry for the delay.